### ISSUE TITLE
Crumbling Tiles - Mappings

### ISSUE THREAD

**Issue by radioshadow** [Nasina7/puyodisasm#5]
I'm not sure what to call this effect, but it's the part where the bottom tiles crumble after all the Puyo have reached the top of the screen:
![PuyoPuyo_002](https://github.com/Nasina7/puyodisasm/assets/62575054/e6a408f7-22bd-44f5-b2a1-71413138c626)

Feel free to use a better term when referring to it in the code. Anyway, back in November, I figured out where the crumbling tile mappings were, and today, I figured out what the settings do. See my notes below:

![Mapping1](https://github.com/Nasina7/puyodisasm/assets/62575054/6e660793-f3a2-4d1d-9f66-a40e2de7c345)
![Mapping2](https://github.com/Nasina7/puyodisasm/assets/62575054/6832a049-b8dd-4c91-bdff-99a99202f45c)

I don't know why the mapping data only uses one pointer. I would recommend setting it up like so:

```
Grass_Crumble_Mapping:
          incbin 	"art\bgMappings\crumble\grass\frame_1.bin"
          incbin 	"art\bgMappings\crumble\grass\frame_2.bin"
          incbin 	"art\bgMappings\crumble\grass\frame_3.bin"
          incbin 	"art\bgMappings\crumble\grass\frame_4.bin"
          incbin 	"art\bgMappings\crumble\grass\frame_5.bin"
          incbin 	"art\bgMappings\crumble\grass\frame_1.bin"
          incbin 	"art\bgMappings\crumble\grass\board_top.bin"
```

Enjoy!

**Comment by Nasina7** [comment]
Nice find!  I'll implement this tomorrow. :+1:

**Comment by Nasina7** [comment]
Implemented in ff0f228 (and 1fb6215 because I forgot to remove a few files before pushing the changes to Github :p ).
